applications/minotari_merge_mining_proxy/src/config.rs

```diff
@@ -177,7 +177,7 @@ impl Default for MergeMiningProxyConfig {
             range_proof_type: RangeProofType::RevealedValue,
             p2pool_enabled: false,
             monerod_fallback: Default::default(),
-            monerod_connection_timeout: Duration::from_secs(2),
+            monerod_connection_timeout: Duration::from_secs(105),
```

Comment:
This seems huge, especially for the default value - I would only go as high as 5s. Where only a single monerod server is assigned in `monerod_url` (_non_default use like in Tari Universe_), this big value could be assigned during start-up.